ballerina/tests/generated_client.bal

```diff
@@ -197,6 +210,36 @@ client class RainierClient {
         return 'object;
     };
 
+    isolated resource function get orderItems() returns stream<OrderItem, error?> {
```

Comment:
Shall we also change the resource names to lowercase `orderitem`

Reply:
Fixed in [775f43e](https://github.com/ballerina-platform/module-ballerina-persist/pull/105/commits/775f43eb14f46d103a58633e1e781921f7222a05)